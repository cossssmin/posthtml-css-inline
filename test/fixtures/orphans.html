<!DOCTYPE html>
<html>
<head>
  <style>
    .flex {
      display: flex;
    }

    .m-0 {
      margin: 0;
    }

    @media screen {
      .flex {
        display: flex;
      }
    }
  </style>
</head>
<body>
  <p class="flex">a</p>
</body>
</html>
