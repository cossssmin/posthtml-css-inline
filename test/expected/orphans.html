<!DOCTYPE html>
<html>
<head>
  <style>@media screen {
      .flex {
        display: flex;
      }
    }</style>
</head>
<body>
  <p class="flex" style="display: flex">a</p>
</body>
</html>
